server/gitrest/Dockerfile

```
@@ -2,7 +2,7 @@
 # Licensed under the MIT License.
 # DisableDockerDetector "No feasible secure solution for OSS repos yet"
 
-FROM node:14.19.1-stretch AS base
+FROM node:18.17.1-buster-slim AS base
```

if you don't want to update node as part of this:
```suggestion
FROM node:14.19.1-buster-slim AS base
```

Not sure if that works: I don't use docker.

If there is no buster-slim image with node 14, we might need to take a different approach.

---

Can we just do this change in the other PR? I suspect that for the other PR we should use node 18 here right?